inc/redirect/namespace.php

```diff
@@ -208,7 +212,7 @@ function rewrite_rules_for_open() {
  * @copyright 2014 Brad Payne
  * @since 3.8.0
  */
-function do_open() {
+function do_open( $do_download = null ) {
```

Comment:
Added a callback function to be able to swap download implementation for testing purposes.